web/src/lib/utils/asset-utils.ts

```diff
@@ -102,14 +102,14 @@ export const downloadFile = async (asset: AssetResponseDto) => {
   }
   const assets = [
     {
-      filename: `${asset.originalFileName}.${getFilenameExtension(asset.originalPath)}`,
+      filename: `${asset.originalFileName}`,
```

Comment:
You can just pass `asset.originalFileName` now without the `${}` stuff (below as well)

Reply:
good catch